lib/moab/storage_object_validator.rb

```diff
@@ -49,6 +51,7 @@ def self.error_code_to_messages
           NO_SIGNATURE_CATALOG => "Version: %{addl} Missing signatureCatalog.xml",
           NO_MANIFEST_INVENTORY => "Version: %{addl} Missing manifestInventory.xml",
           NO_XML_FILES => "Version: %{addl} Missing all required metadata files",
+          DIRS_DETECTED => "Should only contain files, but directories were present",
```

Comment:
it would be good to indicate which directory we are talking about here

Reply:
👍